release/ray_release/test.py

```diff
@@ -414,6 +414,7 @@ def get_byod_runtime_env(self) -> Dict[str, str]:
             # Logs the full stack trace from Ray Data in case of exception,
             # which is useful for debugging failures.
             "RAY_DATA_LOG_INTERNAL_STACK_TRACE_TO_STDOUT": "1",
+            "LD_PRELOAD": "/usr/lib/x86_64-linux-gnu/libjemalloc.so",
```

Comment:
should this be built into the ray container rather than put it here just for testing?